portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/captcha/Captcha.testcase

```diff
@@ -22,7 +22,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-29540."
+	@description = "This is a use case for LRQA-29540. Assert CAPTCHA is enabled when creating an account"
```

Comment:
Please use Verify instead of Assert
cc @weller0246 , @ferenconodiD

Reply:
Description has been changed